cli/team.go

```diff
@@ -0,0 +1,120 @@
+package cli
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+)
+
+// GetTeams - Get Honeybadger Teams
+func (hbc *HoneybadgerClient) GetTeams() ([]HoneybadgerTeam, error) {
+	var hbTeams HoneybadgerTeams
+
+	url := fmt.Sprintf("%s/v2/teams", hbc.HostURL)
+	req, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		return hbTeams.Teams, err
+	}
+
+	body, err := hbc.DoRequest(req)
+	if err != nil {
+		return hbTeams.Teams, err
+	}
+
+	err = json.Unmarshal(body, &hbTeams)
+	if err != nil {
+		return hbTeams.Teams, err
+	}
+
+	return hbTeams.Teams, nil
+}
+
+// FindTeamByName - Find Team by name
+func (hbc *HoneybadgerClient) FindTeamByName(teamName string) (HoneybadgerTeam, error) {
```

Comment:
If you return an object, the object is copied from the `return` command. If the object to be returned is new (created inside the method, as it's in this case), I guess it's better to return just the pointer.

```suggestion
func (hbc *HoneybadgerClient) FindTeamByName(teamName string) (*HoneybadgerTeam, error) {
```

In this way, you can then return just a `nil` on error cases.

Reply:
Good to know. I'll keep in mind 😄